streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/resource/ResourceTypeHandleFactory.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.console.core.service.resource;
+
+import org.apache.streampark.console.base.domain.RestResponse;
+import org.apache.streampark.console.core.entity.Resource;
+import org.apache.streampark.console.core.enums.ResourceTypeEnum;
+import org.apache.streampark.console.core.service.impl.ResourceServiceImpl;
+import org.apache.streampark.console.core.service.resource.impl.AbstractResourceHandle;
+import org.apache.streampark.console.core.service.resource.impl.ConnectorResourceHandle;
+import org.apache.streampark.console.core.service.resource.impl.FlinkAppResourceHandle;
+
+import com.google.common.collect.ImmutableMap;
+
+public class ResourceTypeHandleFactory {
+
+  public static ResourceHandle getResourceHandle(
```

Comment:
ResourceServiceImpl? I think using interfaces ResourceService would be better

Reply:
Thanks for your feedback, it is better to use the interface, I will rethink the design later